### PR TITLE
Merge themes, fix global tab variable

### PR DESCRIPTION
**Pull request by ycaokris**
Merge themes to fix https://github.com/plotly/dash-tektronix-350/issues/1

Deployed on https://dash-playground.plotly.host/dash-daq-tektronix350/

## Problem description
Original app is using global variable to get/store/update running stats and tab number, which on deployment server is saved in memory and shared by different users, as a result, tab number accumulates on DDS and never reset to initial state.

## Changes

- [x] Use dcc.Store to save variables(running stats) shared/updated by callbacks
- [x] Merge light theme and dark theme as one layout to share storaged value
- [x] Initialize tab number as one on page load
- [x] Move stylesheets into assets folder

**Callbacks in this app has been re-arranged so that controller values are saved across themes, functionality remains same with original app**

### CODE REVIEW

**Comment by ycaokris** [comment]
@shammamah  I'd also love to double-check app name before moving into dash-sample-apps. Are we still using `dash-daq-tektronix350` or rename with mentioning `function oscilloscope` ?

**Comment by shammamah** [comment]
@ycaokris Let's keep the name the same.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 A mock app of the Tektronix TDS 350 Oscilloscope. View a demo and learn more about this application from our blog entry [here](https://www.dashdaq.io/tektronix-function-generator) and [here](https://www.dashdaq.io/oscilloscope-logging).
 
+
```
Comment (shammamah) [inline README.md]: Why is this here?

File: requirements.txt [hunk 2]
```diff
@@ -1,12 +1,6 @@
...
-dash-daq==0.1.4
-dash-html-components==0.13.5
-dash-renderer==0.19.0
-dash-table==3.5.0
+dash==0.43.0
 numpy==1.16.1
 scipy==1.2.1
-dash-daq==0.1.4
 flake8
+dash-daq>=0.1.4
```
Comment (shammamah) [inline requirements.txt]: Yay for cleaning this up!

File: README.md (hunk 1 above)
Comment (ycaokris) [reply README.md]: Removed.

**Comment by ycaokris** [comment]
Deployed on gallery : https://dash-gallery.plotly.host/dash-daq-tektronix350/
